src/codegen.cpp

```diff
@@ -6267,7 +6265,7 @@ static void init_julia_llvm_env(Module *m)
                   "Size of jl_array_flags_t is not the same as int16_t");
     Type *jl_array_llvmt =
         StructType::create(jl_LLVMContext,
-                           makeArrayRef(vaelts),
+                           ArrayRef<Type*>(vaelts,sizeof(vaelts)/sizeof(vaelts[0])),
```

Comment:
It would look simpler, but in the end, makes no difference to the compiler, so not important.